Source/Schema.NET/Values{T1,T2,T3,T4}.cs

```diff
@@ -202,6 +222,42 @@ public Values(IEnumerable<object> items)
         public static implicit operator Values<T1, T2, T3, T4>(T4 item) => new Values<T1, T2, T3, T4>(item);
 #pragma warning restore CA2225 // Operator overloads have named alternates
 
+        /// <summary>

```

Comment:
I guess I just forgot to add these...

Reply:
It only came up while I was writing a test for the date/time stuff. I was originally going to have it as a separate PR.

Funny enough though, I don't think my final test cases actually needed that change but I guess I will leave it in now for consistency.